lib/matplotlib/axes/_base.py

```diff
@@ -4754,6 +4759,9 @@ def __init__(self, figure, artists):
             self.figure = figure
             self.artists = artists
 
+        def get_figure(self, **kwargs):
```

Comment:
Do we want to do `**kwargs` here?


The version elsewhere will also accept `root` as a positional arg, so it is not fully compatible with that signature (of course, it isn't actually changing the behavior here)

That said, this is an internal class inside of of a private function, which is only called in one place, and I'm not even convinced calls `get_figure` _ever_... so perhaps not worth being overly prescriptive here.

Reply:
Hmmm.  If we remove this method then this test
`
lib/matplotlib/tests/test_axes.py::test_zorder_and_explicit_rasterization
`
Fails here
https://github.com/matplotlib/matplotlib/blob/7c70bea3c0e88bdd1dd59ecf5f544514d8b6264f/lib/matplotlib/artist.py#L78-L79

Since that uses the root figure, I think we should also be getting the root figure here
https://github.com/matplotlib/matplotlib/blob/7c70bea3c0e88bdd1dd59ecf5f544514d8b6264f/lib/matplotlib/axes/_base.py#L3149-L3150

As for the `**kwargs`, I don't really remember why I chose that, so happy to switch it to just the _root_ keyword.